docs/reference/sql/endpoints/rest.asciidoc

```diff
@@ -106,6 +106,21 @@ s|Description
 
 |===
 
+The `CSV` format accepts the following additional formatting URL query attributes:
+
+* `header`
++
+Controls if the CSV header should be included in the response or not. It can take one of the following values:
++
+   - `present` (default): include the header;
+   - `absent`: exclude the header.
++
+* `delimiter` (default: `,`)
++
+Indicates which character should be used to separate the CSV values. The following values cannot be used for this attribute:
+apostrophe (`"`), carriage-return (`\r`) and new-line (`\n`). The tab (`\t`) can also not be used, the `tsv` format needs to be used
```

Comment:
Isn't it better to say `double quote`  instead of `apostrophe` ?

Reply:
Oops, you're right, thanks. Fixed.